lib/puppet/util/windows/service.rb

```diff
@@ -331,6 +331,30 @@ def stop(service_name)
     end
     module_function :stop
 
+    # Resume a paused windows service
+    #
+    # @param [:string] service_name name of the service to resume
```

Comment:
I believe the correct rdoc format is `@param [String] service_name`

Reply:
Thanks for pointing that out. It looks like we may need a separate PR to update the rdoc comments b/c there's some other formatting errors in the module (e.g. [here](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/service.rb#L258))